docs/INSTALL.md

```diff
@@ -136,7 +136,7 @@ the following commands for your OS:
 <details>
   <summary>macOS</summary>
   
-  First, install [Homebrew](https://brew.sh) if you don‘t already have it.
+  First, install [Homebrew](https://brew.sh) if you don’t already have it.
```

Comment:
nit: just stumbled over this, in the other place you used `don't`, here `don’t`, where the first one is correct, I think